src/algebra/group_action_hom.lean

```diff
@@ -103,8 +115,22 @@ def to_quotient : G →[G] quotient_group.quotient H :=
 
 end mul_action_hom
 
+-- TODO Is this _really_ missing!!? Where should it go?
+/-- The inverse of a bijective `monoid_hom` is a `monoid_hom`. -/
+@[to_additive "The inverse of a bijective `add_monoid_hom` is an `add_monoid_hom`."]
```

Comment:
Surely I'm missing the obvious. This _has_ to exist already but I couldn't find it!